src/docs/sphinx/quickstart.rst

```diff
@@ -291,6 +291,8 @@ If you plan to install the developer tools, you should also run
    $ ln -s /opt/homebrew/opt/llvm@14/bin/clang-format /opt/homebrew/bin/clang-format
 
 If you have installed Homebrew using the default installation prefix, most packages will be accessible through the prefix ``/opt/homebrew``.
+Note for Intel-based Macs, the installation prefix is ``/usr/local``. If you set a custom prefix or aren't sure what the prefix is, run ``brew --prefix``.
```

Comment:
@chapman39, since you asked, my preference would be to replace `/opt/homebrew` everywhere in this section with `$(brew --prefix)`  since that will always do the right thing. But we can do that later. Let's just get this in.